server/scripts/20220722000000-backfill-subscriptions.test.ts

```diff
@@ -0,0 +1,151 @@
+import { Event } from "@server/models";
+import { buildDocument, buildUser } from "@server/test/factories";
+import { flushdb } from "@server/test/support";
+import script from "./20220722000000-backfill-subscriptions";
+
+beforeEach(() => flushdb());
+
+describe("#work", () => {
+  it("should create subscriptions and events for document creator and collaborators", async () => {
```

Comment:
This test is a bit convoluted, but these cyclic dependencies help solve a couple of edge cases.